.ci/azure/linux.yml

```diff
@@ -167,15 +168,30 @@ jobs:
   - script: |
       python3 -m pip install --user virtualenv
       python3 -m virtualenv -p /usr/bin/python3.8 .env3
-      source .env3/bin/activate
+      source .env3/bin/activate && source $(SETUPVARS)
       python -m pip install --upgrade pip
       python -m pip install -r $(REPO_DIR)/modules/custom_operations/tests/requirements.txt
       python -m pip install $(INSTALL_DIR)/tools/openvino-*.whl
       python -m pip install $(OPENVINO_REPO_DIR)/tools/mo/
-      python -m pip install $(REPO_DIR)/modules/custom_operations/user_ie_extensions/tokenizer/python/.[all]
     workingDirectory: $(WORK_DIR)
     displayName: 'Create virtual env'
 
+  - script: |
+      source $(WORK_DIR)/.env3/bin/activate && source $(SETUPVARS)
+      python -m pip install build
+      python -m build --wheel --outdir $(BUILD_WHEEL_DIR) $(REPO_DIR)/modules/custom_operations
+    workingDirectory: $(WORK_DIR)
+    displayName: 'Build tokenizers wheel'
+
+  - script: |
+      source $(WORK_DIR)/.env3/bin/activate
+      python -m pip install $(BUILD_WHEEL_DIR)/*.whl
```

Comment:
should we keep `[all]` ?
Looks like currently extras are missed

Reply:
fixed